### ISSUE TITLE
Set the world view initially instead of a delayed zoom in

### ISSUE THREAD

**Issue by mourner** [bmcbride/leaflet-users-map#1]
Hi! As there are markers all over the world now, you can set the map view to the whole world initially (e.g. with `map.fitWorld()`) instead of zooming after the page gets all the markers.

**Comment by bmcbride** [comment]
True, but I did want to keep the code generic enough for use on other projects, which may not cover a global extent. The firstLoad check is a bit of a hack, but it gets the job done.
